config.go

```diff
@@ -959,12 +961,19 @@ func (c *ConfigTLS) InterpolateMonitor(m *ConfigMonitor) {
 }
 
 func (c ConfigTLS) LoadTLS() (*tls.Config, error) {
+
+	tlsConfig := &tls.Config{}
+
+	// Skip TLS verification
+	if c.SkipVerify != nil && *c.SkipVerify {
```

Comment:
`blip.True()`

Reply:
Move this block under `if c.Cert == "" && c.Key == "" && c.CA == "" {`.  This func is supposed to return `nil, nil` if there's no TLS config.  So skip-verify logic needs to happen only if there is _some_ TLS config.

Reply:
so you mean
```
tls:
  skip-verify: true
  ``` 
  is invalid because they CA or {cert,key} isn't supplied?
  
  So a valid skip verify config is only one of the below?
```
tls:
  skip-verify: true
  ca: path_to_ca
  ```
or 
```
tls:
    skip-verify: true
     cert: path_to_cert
     key: path_to_key
     ```

Reply:
Yes, that's correct.

Reply:
Updated, but any strong reason why we mandate those even if we are asking server to ignore certs/CA anyway?